app/qml/main.qml

```diff
@@ -7,7 +7,6 @@ import QtQuick.Controls 2.2
 import QtMultimedia 5.8
 import QtQml.Models 2.2
 import QtPositioning 5.8
-import QtLocation 5.8
```

Comment:
note that in iOS the resolving of the QtQuick import is not working properly. So everything that is imported from QgisQuick library MUST be also included in the mail.qml

Reply:
Sure but I can run Input even without this line (so it does not seem to be needed), moreover this import has been added very recently (1-2 weeks ago)

Reply:
(I wanted to say I do not have that module installed on my linux system at all)